static/locales/en-US.yaml

```diff
@@ -406,11 +406,16 @@ Settings:
   Player Settings:
     Player Settings: Player
     Play Next Video: Play Next Video
-    Turn on Subtitles by Default: Turn on Subtitles by Default
+    Turn on Subtitles by Default: Enable Subtitles by Default
     Autoplay Videos: Autoplay Videos
     Proxy Videos Through Invidious: Proxy Videos Through Invidious
     Autoplay Playlists: Autoplay Playlists
-    Enable Theatre Mode by Default: Enable Theatre Mode by Default
+    Default Viewing Mode:
```

Comment:
I dont see Full Window in here. How does that get translated?

Reply:
That's the one. I thought it would be better to reuse the existing label than create a new duplicate one. We do this kind of thing in a handful of other places in the app as well, and I suppose a better practice would probably be to move all such keys to a `Global` section like was done in #6495, most likely in a separate effort.